introduction/it/05-principles.asciidoc

```diff
@@ -0,0 +1,66 @@
+
+* Il progetto/repository e la sua direzione
+* Requisiti di funzionalità eccezionali
+* I progressi sui requisiti delle funzionalità
+* Il processo decizionale dell'host team
+
+Quanto possibile, quanto descritto sopra dovrebbe essere comunicato chiaramente ed in dettaglio, dalle definizioni interne degli elementi dei team a specifici scenari di casi speciali del progetto.
+Questa comunicazione dovrebbe essere fatta in maniera tale che possa essere facilmente ricercata e compresa a tutti quelli che non fanno parte del core team.
+
+=== Tutoraggio prioritizzato
+
+Il _tutoraggio_ dall'host team al guest team attraverso i https://innersourcecommons.org/resources/learningpath/trusted-committer/index[trusted committers] è un aspetto chiave dell'InnerSource.
+I https://innersourcecommons.org/resources/learningpath/contributor/index[Contributors] nel guest team vengono allineati contestualmente in modo tale da far loro capire abbastanza il progetto/repository dell'host team al fine di poterlo modificare con successo.
+Nel fare ciò, arrivano a capire meglio il sistema software dell'host team come un utente e ambasciatore del progetto/software.
+Questo singolo contributore può, con il passare del tempo e con l'esperienza, assumere un ruolo più alto nel progegtto com quello di trusted committer.
+
+E' fondamentale che questo tutoraggio per i contributori sia _Prioritizzato_ dall'host team.
+L'host team dovrebbe sforzarsi di dedicare del tempo per fare da mentore ai contributori del guest team _nel momento in cui il contributore ne ha bisogno_ al contrario invece di quando è conveniente per l'host team.
+A volte potrebbe richiedere un cambiamento culturare per gli ingegneri dell'host team investire del tempo aiutando gli altri a scrivere codice piuttosto che scrivere codice da soli.
+Questo tutoraggio ha un valore per entrabi i singoli contributori che per l'host team, e vale la pena farlo bene.
+Si rivela reciprocamente vantaggioso nel lungo periodo. Migliorando il codice, il contributore forgia o migliora le relazioni all'interno dell'organizzazione che altrimenti non sarebbero potute esistere.
+L'open source riconosce prontamente questo punto e considera un onore ottenere lo stato di trusted committer su un progetto.
+
+=== Contribuzione volontaria al codice
+La seconda parola _volontaria_ significa l'impegno nell'InnerSource da entrambi guest e host team avviene di loro spontanea volontà.
+Il guest team dona volontariamente il codice all'host team e l'host team volontariamente lo accetta.
+Questa natura di partecipazione significa che ogni team deve essere sicuro che il loro coinvolgimento aggiunga valore agli obiettivi degli altri.
+Mai sarà richiesto ad un host team di accettare una contribuzione che non sia in linea con la missione complessiva.
+Mai sarà richiesto ad un guest team di inviare un contribuzione che alla fine non favorisce la loro missione e le loro priorità.
```

Comment:
More than `contribuzione` I'd say `contributo`, also look at `un contribuzione`, I think you missed an `a` :D